crates/astria-sequencer/justfile

```diff
@@ -28,18 +28,18 @@ run-testnet:
 
 [linux]
 stop-testnet:
-  pkill cometbft && pkill astria-sequencer
+  pkill cometbft && pkill -f astria-sequencer
 [linux] 
 run-testnet: 
   cargo build
-  ${OUT_DIR} ${NUM_VALIDATORS} ./scripts/testnet.sh
+  ./scripts/testnet.sh
```

Comment:
With this change, the `[macos]` and `[linux]` versions are identical, so could probably be deduplicated?

Reply:
changed so the linux version is different